compiler/bindings/python/IREECompilerDialectsModule.cpp

```diff
@@ -693,4 +694,80 @@ NB_MODULE(_ireeCompilerDialects, m) {
       "isa_attention_op", &ireeCodegenMlirOperationIsACodegenAttentionOp,
       "Checks if the given operation is an IREE LinalgExt attention op.",
       py::arg("op"));
+
+  //===-------------------------------------------------------------------===//
+  // Binding to utility function ireeCodegenGetIGEMMGenericConvDetails
+  //===-------------------------------------------------------------------===//
+  py::class_<ireeCodegenIGEMMGenericConvDetails>(iree_codegen_module,
+                                                 "IGEMMGenericConvDetails")
+      .def_prop_ro("igemm_contraction_maps",
+                   [](const ireeCodegenIGEMMGenericConvDetails &self) {
+                     return self.igemmContractionMaps;
+                   })
+      .def_prop_ro("igemm_loop_bounds",
+                   [](const ireeCodegenIGEMMGenericConvDetails &self) {
+                     return getIntArrayAttrValues(self.igemmLoopBounds);
+                   })
+      .def_prop_ro("igemm_loop_iterators",
+                   [](const ireeCodegenIGEMMGenericConvDetails &self) {
+                     return self.igemmLoopIterators;
+                   })
+      .def_prop_ro("im2col_output_perm",
+                   [](const ireeCodegenIGEMMGenericConvDetails &self) {
+                     return getIntArrayAttrValues(self.im2colOutputPerm);
+                   })
+      .def_prop_ro(
+          "filter_reassoc_indices",
+          [](const ireeCodegenIGEMMGenericConvDetails &self)
+              -> std::vector<std::vector<int64_t>> {
+            std::vector<std::vector<int64_t>> result;
+            MlirAttribute attr = self.filterReassocIndices;
+            assert(!mlirAttributeIsNull(attr) && mlirAttributeIsAArray(attr) &&
+                   "filterReassocIndices should be a valid ArrayAttr");
+            size_t n = mlirArrayAttrGetNumElements(attr);
+            result.reserve(n);
```

Comment:
```suggestion
            MlirAttribute attr = self.filterReassocIndices;
            assert(!mlirAttributeIsNull(attr) && mlirAttributeIsAArray(attr) &&
                   "filterReassocIndices should be a valid ArrayAttr");
            size_t n = mlirArrayAttrGetNumElements(attr);
            std::vector<std::vector<int64_t>> result;
            result.reserve(n);
```
try to keep the definition close to the first use